### DISCUSSION TITLE
md previewed bug

### DISCUSSION THREAD

**Discussion by yfg2014** [community/community#48134] [Mobile] [answered]
### Select Topic Area

Bug

### Body

An md file named in Chinese, while the content contains Chinese content, cannot be previewed on the mobile app。please check it .if can fix it , it will be good .

**Comment by stevepopovich** [comment, marked as answer]
Hi @yfg2014, do you have a link where this is happening? Thank you!

**Reply by yfg2014** [reply]
https://github.com/yfg2014/noteScript/blob/main/%E4%B8%AD%E6%96%87%E6%B5%8B%E8%AF%95.md ,  hi,you can use this link . my test platform   is  apple mobile .

**Reply by yfg2014** [reply]
This md file is named in Chinese and contains Chinese content, it can be opened in the browser, but cannot be opened in the GitHub APP, it will show that it cannot be previewed.

**Reply by stevepopovich** [reply]
I can reproduce the issue! Interesting. I reported this to the team to get fixed. Thank you!

**Reply by yfg2014** [reply]
Thank you very much!

**Reply by stevepopovich** [reply]
Hi @yfg2014 is this still happening for you? I can't reproduce the issue anymore and I am not sure what changed 🤔

**Reply by L-Super** [reply]
> Hi @yfg2014 is this still happening for you? I can't reproduce the issue anymore and I am not sure what changed 🤔 

Me too.It's fixed？

**Reply by yfg2014** [reply]
hi,@stevepopovich , The preview is normal on my side too. Didn't your team fix it?

**Reply by stevepopovich** [reply]
I am not sure exactly what changed. Could of been a temporary server blip. We will treat this as resolved for now. Let me know if this breaks again!

**Comment by L-Super** [comment]
Me too.

**Comment by Jibggjii** [comment]
👏
